File: src/main/java-graphql/io/github/graphql/model/UnlinkRepositoryFromProjectMutationResponse.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class UnlinkRepositoryFromProjectMutationResponse extends GraphQLResult<Map<String, UnlinkRepositoryFromProjectPayloadTO>> {

    private static final String OPERATION_NAME = "unlinkRepositoryFromProject";

    public UnlinkRepositoryFromProjectMutationResponse() {
    }

    public UnlinkRepositoryFromProjectPayloadTO unlinkRepositoryFromProject() {
        Map<String, UnlinkRepositoryFromProjectPayloadTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/kotlin-graphql/io/github/graphql/model/UnlinkRepositoryFromProjectMutationResponse.kt
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult

@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
open class UnlinkRepositoryFromProjectMutationResponse : GraphQLResult<MutableMap<String, UnlinkRepositoryFromProjectPayloadTO?>>() {

    companion object {
        const val OPERATION_NAME: String = "unlinkRepositoryFromProject"
    }

    fun unlinkRepositoryFromProject(): UnlinkRepositoryFromProjectPayloadTO? {
        val data: MutableMap<String, UnlinkRepositoryFromProjectPayloadTO?> = super.getData()
        return data[OPERATION_NAME]
    }
}